src/main/java/teammates/ui/template/StudentFeedbackResultsQuestionWithResponses.java

```diff
@@ -1,7 +1,12 @@
 package teammates.ui.template;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+/**
+ * Manages student feedback results questions details, self-responses tables and others-responses tables separately
```

Comment:
Thanks for the input @wkurniawan07 .

As mentioned, we can put `View Model for student/feedbackResults/questionWithResponses.tag`. Your explanation is also fair enough and we can keep it.